docs/examples/plotting/live_plotting.ipynb

```diff
@@ -16,18 +16,18 @@
    "metadata": {},
    "source": [
     "## TL;DR\n",
-    "Enable live plotting by specifying 'subscripition.default_subscriber' in your qcodesrc.json."
+    "Enable live plotting by specifying `subscripition.default_subscriber` in your `qcodesrc.json`."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "## Introduction\n",
-    "Live plotting in qcodes can be achieved by sending the acquired data to other applications. For this an adapter for the specific application is necessary. These adapter register as subscribers to the qcodes dataset and expose a callback that gets called on regular intervals that can be specified.\n",
+    "Live plotting in QCoDeS can be achieved by sending the acquired data to other applications. To be able achieve this communication, an adapter for the specific application is necessary. These adapter register as subscribers to the QCoDeS `dataset` and expose a callback that gets called on regular intervals that can be specified.\n",
     "\n",
     "## Live plotting with plottr as an example\n",
-    "An example of a live plotting application is for example [*plottr*](https://github.com/kouwenhovenlab/plottr). After installing *plottr* you can use it by simply setting `subscription.default_subscribers` to `[\"Plottr\"]` in your `qcodesrc.json` file or alternatively by updating your qcodes config for your current session:"
+    "An example of a live plotting application is for example [*plottr*](https://github.com/kouwenhovenlab/plottr). After installing *plottr* you can use it by simply setting `subscription.default_subscribers` to `[\"Plottr\"]` in your `qcodesrc.json` file or alternatively by updating your QCoDeS config for your current session:"
```

Comment:
btw, this section is wrong. plottr doesn't request the subscribers to work. but perhaps that's for another PR.

Reply:
Sure, I have never used plottr so I have simply accepted what is written as the truth.